src/timeout.py

```diff
@@ -130,8 +134,7 @@ class cmd_run(Command):
     ]
 
     def __call__(self, configs, args):
-        timeout = Timeout(configs, args)
-        timeout.run()
+        Timeout(configs, args).run()
```

Comment:
```suggestion
        return Timeout(configs, args).run()
```

Reply:
That was actually fixed in #175 but I've fixed it here now anyway.